packages/cli/src/commands/setup/i18n/i18n.js

```diff
@@ -0,0 +1,90 @@
+import fs from 'fs'
+import path from 'path'
+
+import Listr from 'listr'
+import execa from 'execa'
+import chalk from 'chalk'
+
+import c from 'src/lib/colors'
+import { getPaths, writeFile } from 'src/lib'
+
+export const command = 'i18n'
+export const description = 'Setup i18n'
+export const builder = (yargs) => {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing configuration',
+    type: 'boolean',
+  })
+}
+
+const INDEX_JS_PATH = path.join(getPaths().web.src, 'index.js')
+
+export const handler = async ({ force }) => {
+  const tasks = new Listr([
+    {
+      title: 'Installing packages...',
+      task: async () => {
+        await execa('yarn', [
+          'workspace',
+          'web',
+          'add',
+          'i18n',
+          'i18next',
+          'i18next-browser-languagedetector',
+          'i18next-http-backend',
+          'react-i18next'
+        ])
+      },
+    },
+    {
+      title: 'Configuring i18n...',
+      task: () => {
+        /**
+         * Write i18n.js in web/src
+         */
+        return writeFile(
+          getPaths().web.src,
```

Comment:
This path seems incorrect? Should it not be something  like `path.join(paths.web.src, i18n.js)`